.github/workflows/auto_format.yaml

```
@@ -1,102 +1,110 @@
-          skip_checkout: true
+            - name: Check for changes
+              id: check-changes
+              run: |
+                  if [[ -n "$(git status --porcelain)" ]]; then
+                    echo "changes-detected=true" >> $GITHUB_OUTPUT
+                    echo "📝 Formatting changes detected"
+                  else
+                    echo "changes-detected=false" >> $GITHUB_OUTPUT
+                    echo "✅ No formatting changes needed"
+                  fi
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)